cms/templates/js/self-paced-due-date-editor.underscore

```diff
@@ -0,0 +1,16 @@
+<ul class="list-fields list-input datepair date-setter">
```

Comment:
What is the `datepair` class?

Reply:
I think it's for date inputs that have both the number date and time input! But going to remove it here since I'm only working with a single number

Reply:
So a lot of times with class names, they will be used as selectors in JS or more likely for styling in a CSS or SCSS file. I would recommend searching those file types to see if you can find out where/how they are used to confirm, but I suspect your hunch is right :)